### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by mirandaweston**

### CODE REVIEW

File: frontend/cypress/support/component.js [hunk 1]
```diff
@@ -28,3 +28,6 @@ Cypress.Commands.add("mount", mount);
 
 // Example use:
 // cy.mount(<MyComponent />)
+
```
Comment (joel-powell) [inline frontend/cypress/support/component.js]: can we remove this duplicate please?

File: frontend/package-lock.json [hunk 2]
```diff
@@ -16,9 +16,12 @@
         "axios": "^1.3.4",
         "axios-hooks": "^4.0.0",
         "clsx": "^1.2.1",
+        "dotenv": "^16.0.3",
```
Comment (joel-powell) [inline frontend/package-lock.json]: can we remove dotenv please:

```
cd frontend
npm uninstall dotenv
```

File: frontend/public/env.js [hunk 3]
```diff
@@ -0,0 +1 @@
+window.env = {};
```
Comment (joel-powell) [inline frontend/public/env.js]: do we need this file?

File: frontend/public/index.html [hunk 4]
```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <script id="react-dotenv" src="./env.js"></script>
```
Comment (joel-powell) [inline frontend/public/index.html]: do we need this line?

File: frontend/src/components/Home.cy.js
Comment (joel-powell) [inline frontend/src/components/Home.cy.js:1]: can we please move this file into the home folder with the component

File: frontend/src/components/home/Home.js [hunk 5]
```diff
@@ -21,10 +34,13 @@ const Home = () => {
...
 
-          <div className="flex flex-1 justify-end">
-            <div className="text-sm font-semibold leading-6 text-white">
-              Log in <span aria-hidden="true">&rarr;</span>
-            </div>
+          <div className="flex flex-1 justify-end space-x-2">
+            <Link to="/login" className="text-sm font-semibold leading-6 text-white">
+              Log in
+            </Link>
+            <Link to="/register" className="text-sm font-semibold leading-6 text-white">
```
Comment (joel-powell) [inline frontend/src/components/home/Home.js]: /register is now /signup

**Review by joel-powell — CHANGES_REQUESTED** [review]
As well as these fixes, please can we also run through the code with prettier and eslint

**Review by joel-powell — APPROVED** [review]

**Review by doragores — APPROVED** [review]

**Pull request merged**